docs/maintaining/index.md

```diff
@@ -8,4 +8,5 @@
    overview
    supported-python-versions
    making-a-release
+   testing
```

Comment:
I would argue that this isn't for maintainers, but instead it is for contributors.

Perhaps the contributing docs need breaking down into a few sections like the maintaining ones are?